core/app/models/spree/shipment.rb

```diff
@@ -255,10 +256,6 @@ def shipped=(value)
       self.shipped_at = Time.current
     end
 
-    def shipping_method
-      selected_shipping_rate.try(:shipping_method) || shipping_rates.first.try(:shipping_method)
```

Comment:
This would be a good idea but will remove the default value (shipping_rates.first.try(:shipping_method)).